Firestore/Source/Core/FSTSyncEngine.mm

```diff
@@ -523,15 +524,14 @@ - (void)emitNewSnapshotsAndNotifyLocalStoreWithChanges:(const MaybeDocumentMap &
 
         if (viewChange.snapshot.has_value()) {
           newSnapshots.push_back(viewChange.snapshot.value());
-          FSTLocalViewChanges *docChanges =
-              [FSTLocalViewChanges changesForViewSnapshot:viewChange.snapshot.value()
-                                             withTargetID:queryView.targetID];
-          [documentChangesInAllViews addObject:docChanges];
+          LocalViewChanges docChanges =
+              LocalViewChanges::MakeChanges(viewChange.snapshot.value(), queryView.targetID);
+          documentChangesInAllViews.push_back(docChanges);
```

Comment:
Nit: you can `std::move` `docChanges` into the vector for additional efficiency.

Reply:
Done.